src/aiida/orm/implementation/storage_backend.py

```diff
@@ -471,6 +471,22 @@ def get_orm_entities(self, detailed: bool = False) -> dict:
         count = QueryBuilder(self).append(Node).count()
         data['Nodes'] = {'count': count}
         if detailed:
+            first_time = (
+                QueryBuilder(self)
+                .append(Node, project=['ctime'], tag='node')
+                .order_by({'node': {'ctime': 'asc'}})
+                .first(flat=True)
+            )
+            last_time = (
+                QueryBuilder(self)
+                .append(Node, project=['ctime'], tag='node')
+                .order_by({'node': {'ctime': 'desc'}})
+                .first(flat=True)
+            )
+
+            data['Nodes']['first_created'] = str(first_time) if first_time else None
```

Comment:
Passing None if there are no Nodes results in the following output from `verdi storage info --detailed`

```
  Nodes:
    count: 0
    first_created: null
    last_created: null
    node_types: []
    process_types: []
```

an alternative would be to pass en empty string, which would then look like this:

```
  Nodes:
    count: 0
    first_created: ''
    last_created: ''
    node_types: []
    process_types: []
```

I think `None` makes sense here. cc @GeigerJ2